runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -52,7 +52,6 @@ protected ILogData removeFromQueue(NavigableSet<Long> queue) {
             }
 
             thisRead = queue.pollFirst();
```

Comment:
Regarding my comment in another PR on whether we were still able to support backpointer views or not, if the answer is no, we might think of removing it. Anyways it was still present as backup when migrating to address maps.

Reply:
I think backpointer view is not compatible with the new mechanism of GC unless the LogUnit servers modify the backpointers during compaction which is fragile and error-prone.

Reply:
I will remove backpointer view.